### PR TITLE
Adds a technical report

### PR DESCRIPTION
**Pull request by nkcr**
[Report.pdf](https://github.com/dedis/dela/files/6466245/Report.pdf)

### CODE REVIEW

File: core/execution/benchmark/Report.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+2. **Simple crypto**: in this series of experiment the smart contract performs a
+   simple operation on ed25519 elliptic curves. The smart contract takes two
+   points (A,B) and a scalar (S) in argument. It then performs (A+B)*S.
+
+## Results
+
+|   [ns/op]    |Native|Unikernel |TCP   |Solidity|Solidity TCP|
+|--------------|------|----------|------|--------|------------|
+|Increment     |0.000 |0.470     |0.014 |0.001   |0.014       |
+|Simple crypto |0.005 |          |      |0.237   |            |
```
Comment (pierluca) [inline core/execution/benchmark/Report.md:123]: Effectively, we can consider that the "increment" implementation is as good as testing the networking overhead. I don't see a need to measure them separately for now.

File: core/execution/benchmark/Report.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## Experiments
+
+We perform two series of experiment:
+
+1. **Increment**: in this series of experiment the smart contract increments
+   the input it receives by 1.
+2. **Simple crypto**: in this series of experiment the smart contract performs a
+   simple operation on ed25519 elliptic curves. The smart contract takes two
+   points (A,B) and a scalar (S) in argument. It then performs (A+B)*S.
```
Comment (gnarula) [inline core/execution/benchmark/Report.md:116]:
```suggestion
   simple operation on ed25519 elliptic curves. The smart contract takes a scalar (s) in argument. It then performs `s*G`, where `G` is the ed25519 base point.
```

**Review by gnarula — CHANGES_REQUESTED** [review]
Looks good! Just need to update the curve operation we do in the contract.

**Review by gnarula — APPROVED** [review]
LGTM

**Pull request merged**